sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -36,6 +37,24 @@
 _LOGGER = logging.getLogger(__name__)
 yaml_provider.fix_pycallable()
 
+try:
+  import jsonschema
+except ImportError:
+  jsonschema = None
+
+if jsonschema is not None:
+  with open(os.path.join(os.path.dirname(__file__),
```

Comment:
The PythonDocs PreCommit does not like the pipeline.schema.yaml file read here (https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Commit/13751/console)

Reply:
Added as a data file.